src/Symfony/Component/Form/Extension/Core/DataTransformer/NumberToLocalizedStringTransformer.php

```diff
@@ -93,13 +93,16 @@ class NumberToLocalizedStringTransformer implements DataTransformerInterface
      */
     const ROUND_HALFDOWN = \NumberFormatter::ROUND_HALFDOWN;
 
+    /*
```

Comment:
Not sure about the deprecation at all. The class is meant to be extended (and also is extended in symfony itself). This why they are protected. It's just __currently__ the case, that the properties are not needed to be read in subclasses. But making them private makes this impossible.